addons/l10n_in/models/template_in.py

```diff
@@ -47,3 +48,61 @@ def _get_in_account_cash_rounding(self):
                 'loss_account_id': 'p213201',
             }
         }
+
+    @template('in', 'account.fiscal.position')
+    def _get_in_account_fiscal_position(self):
+        company = self.env.company
+        state_ids = [Command.set(company.state_id.ids)] if company.state_id else False
+        intra_state_name = company.state_id and _('Within %s', company.state_id.name) or _('Intra State')
+        state_specific = {
+            'fiscal_position_in_intra_state': {
+                'name': intra_state_name,
+                'sequence': 1,
+                'auto_apply': True,
+                'state_ids': state_ids,
+                'country_id': self.env.ref('base.in').id,
+            },
+            'fiscal_position_in_inter_state': {
+                'name': _('Inter State'),
+                'sequence': 2,
+                'auto_apply': True,
+                'tax_ids': self._get_l10n_in_fiscal_tax_vals(),
+                'country_id': self.env.ref('base.in').id
+            },
+        }
+        if company.parent_id:
+            return state_specific
+        return {
+            **state_specific,
+            'fiscal_position_in_outside_in': {
+                'name': _('Outside India'),
+                'sequence': 3,
+                'auto_apply': True,
+                'tax_ids': self._get_l10n_in_fiscal_tax_vals(),
+            },
+            'fiscal_position_in_sez': {
+                'name': _('Special Economic Zone (SEZ)'),
+                'sequence': 4,
+                'tax_ids': self._get_l10n_in_fiscal_tax_vals(),
+            },
```

Comment:
What's the difference between those 2 fiscal positions? (except the auto apply)

Reply:
I suppose it is difficult to have a criterium in auto_apply for the Special Economic Zone cases.  @hamo-odoo

Reply:
About Outside India:
Transaction occurring with foreign entity

About SEZ:
A Special Economic Zone (SEZ) is a geographical region that has economic laws that are more liberal than a country's domestic economic laws.

SEZs in India are areas that offer incentives to resident businesses. SEZs typically offer competitive infrastructure, duty free exports, tax incentives, and other measures designed to make it easier to conduct business. Accordingly, SEZs in India are a popular investment destination for many multinationals, particularly exporters.

cc- @ajjo-odoo

Reply:
> I suppose it is difficult to have a criterium in auto_apply for the Special Economic Zone cases. @hamo-odoo

We don't have any plan to do it as of now (we will only end up making things complex)